docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.7"
+services:
+
+  thrift:
```

Comment:
Maybe `thrift` isn't the best name. I'd rather call it Spark, or `Spark2`. It would be great if we can test DBT against Spark 2 and 3 in the future :-)

Reply:
I changed the names to be more specific